Math/NumberTheory/GCD.hs

```diff
@@ -252,6 +261,31 @@ cw16 (W16# x#) (W16# y#) = coprimeWord# x# y#
 cw32 :: Word32 -> Word32 -> Bool
 cw32 (W32# x#) (W32# y#) = coprimeWord# x# y#
 
+
+newtype Coprimes a b = Coprimes { unCoprimes :: Map.Map a b } deriving (Eq, Show)
+
+singleton :: Ord a => a -> b -> Coprimes a b
+singleton a b = Coprimes (Map.singleton a b)
```

Comment:
Actually, I looked into documentation and it appeared that `Map.singleton` does not require `Ord` constraint. Consequently, we do not need it both here and in two occurrences in `Math.NumberTheory.Prefactored`.